src/OrbitCaptureClient/ApiEventProcessorTest.cpp

```diff
@@ -24,15 +24,18 @@ namespace {
 // Empty implementation of CaptureListener used as helper to reduce the complexity in subclasses.
 class EmptyCaptureListener : public CaptureListener {
  public:
-  void OnCaptureStarted(ProcessData&&,
-                        absl::flat_hash_map<uint64_t, orbit_grpc_protos::InstrumentedFunction>,
-                        TracepointInfoSet, absl::flat_hash_set<uint64_t>) override {}
+  void OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& /*capture_started*/,
```

Comment:
I like the parameter names here but this is now inconsistent with the other methods. I guess the parameter names should be added to the other methods, too.

Reply:
this is for another change. I do not want to add too many "drive by" changes to this one.